addons/minio/2020-01-25T02-50-51Z/install.sh

```diff
@@ -23,6 +33,13 @@ function minio() {
 }
 
 function minio_creds() {
+    if [ -n "$MINIO_HOSTPATH" ]; then
```

Comment:
Is this block safe to run on updates to existing working cluster?

Reply:
This will only execute if a hostPath option is provided, and will delete that `.minio.sys` directory from that hostPath location. So it should be safe. Any particular edge cases you have in mind?

Reply:
I see it also scales the deployment down.  What scales it back up?  And is there a case where we would want to keep the directory?

Reply:
maybe add a comment explaining this logic?

Reply:
this will scale it back up https://github.com/replicatedhq/kURL/pull/1061/files#diff-93b1b6f9d511cb0f57d148dc9967819f594ea155dadadba8182cc43c2914b123R7

Reply:
explanation added